opapi/src/generators/client.ts

```diff
@@ -58,7 +50,7 @@ function generateMethod(props: GenerateClientProps) {
     props,
   )}) => this._innerClient.${operationName}(${generateClientProps(
     props,
-  )}).then((res) => res.data).catch((e) => { throw getError(e) })`
+  )}).then((res) => res.data).catch((e) => { throw errorFrom(e) })`
```

Comment:
This will allow the Axios error to be saved as the "inner error" of an UnknownError, which can then be checked on the client side (e.g. in the CDM frontend, to detect an HTTP 401 without response, to automatically logout the user).